app/views/handlers/thank_you.py

```diff
@@ -23,7 +19,7 @@
 
 class ThankYou:
     DEFAULT_THANK_YOU_TEMPLATE = "thank-you"
-    CENSUS_THANK_YOU_TEMPLATE = "census-thank-you"
+
     PAGE_TITLE = gettext("Thank you for completing the census")
```

Comment:
Think this needs updating

Reply:
This should also be addressed in Pete's PR I believe